### PR TITLE
Merge changes from first round of system-level bench testing (post 8-Jun-20 merge)

### PR DESCRIPTION
**Pull request by disgruntled-patzer**
These are the changes required to the overall system after a first round of system-level bench testing following the pull request exercise on 8 Jun 2020. The first round of testing was meant to catch all the major issues (e.g. syntax errors) that would cause the system to crash if left undetected and hence required an immediate fix.

The list of major changes are:
- Fix two typos in air despatcher: One in the function that publishes to telegram node, and another where incorrect ros param names were passed to the interval variables
- Fix one typo in gnd despatcher, where a message without the required headers were sent to the telegram node
- Add interpreting of message headers in statustext node
- Publish the feedback msgs from SBD node to roslog instead of to the despatchers (which would clutter the screen with "Invalid Message" errors)
- Add OGC SITL and ground launch files
- Add link switching variable for despatcher nodes. Currently they have to be changed manually prior to starting the system

These changes address some of the existing issues here: https://github.com/yonahbox/Yonah_ROS_packages/issues/15

### CODE REVIEW

File: launch/ogc_gndtest.launch [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	<!-- tele args -->
+	<arg name="tdlib_auth_dir" default="/home/lenovo/.tdlib"/>
+	<arg name="monitored_chat" default="tele_link_grp" />
+	<arg name="command_whitelist" default="[Rumesh,Lau]" />
+
+	<!-- sbd args -->
+    <arg name="portID" default="/dev/ttyUSB0"/>
+    <arg name="own_serial" default="12345"/>
+    <arg name="client_serial" default="12345"/>
+	<arg name="credentials" default="$(find sbd)/src/login.txt"/>
```
Comment (danipurwadi) [inline launch/ogc_gndtest.launch]: Just a minor comment I think it is nice to have the lines indented consistently

**Review by danipurwadi — APPROVED** [review]
From what I can see I don't think there are any problems, go ahead with the merge

File: launch/ogc_gndtest.launch [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	<arg name="router_hostname" default="root@192.168.1.1" />
+	
+	<!-- sms args -->
+	<arg name="whitelist" default="$(find sms)/src/whitelist.txt"/>
+	<arg name="client_phone_no" default="'+12345678'"/>
+
+	<!-- tele args -->
+	<arg name="tdlib_auth_dir" default="/home/lenovo/.tdlib"/>
+	<arg name="monitored_chat" default="tele_link_grp" />
+	<arg name="command_whitelist" default="[Rumesh,Lau]" />
```
Comment (rumesh986) [inline launch/ogc_gndtest.launch:17]: Do we want to keep the names?

Comment (disgruntled-patzer) [reply launch/ogc_gndtest.launch:17]: Eventually the names will be removed when your code goes in; and I won't classify usernames as sensitive info, should be ok to keep it here

Comment (rumesh986) [reply launch/ogc_gndtest.launch:17]: Ok then, I did not notice anything else of concern

**Review by rumesh986 — APPROVED** [review]

**Review by huachen24 — APPROVED** [review]

**Pull request merged**